vtex/faststore
# Create Gatsby plugin for OneSignal integration

## What's the purpose of this pull request?

This creates a new package called `@vtex/gatsby-plugin-onesignal`, which enables any store to easily use OneSignal's push notification solution.

## How it works? 

This new plugin adds all necessary setup for OneSignal. Stores should only need a OneSignal `appId` and be good to go after installing this plugin.

For a look into the rendered docs, check: https://github.com/vtex/faststore/blob/gatsby-plugin-onesignal/packages/gatsby-plugin-onesignal/README.md.

## How to test it?

You can access [this](https://sfj-e89c228--storecomponents.preview.vtex.app/apparel---accessories) preview and navigate until you see a notification permission banner. If you see the banner and enable notifications, you're now subscribed and OneSignal is working :).

☝🏻  The preview comes from the PR: https://github.com/vtex-sites/storecomponents.store/pull/803
